src/api/pools/index.js

```diff
@@ -0,0 +1,39 @@
+const { MULTICHAIN_ENDPOINTS } = require('../../constants');
+
+const buildResponse = pools => {
+  return pools.map(pool => {
+    return {
+      id: pool.id,
+      name: pool.name,
+      tokenAddress: pool.tokenAddress,
+      tokenDecimals: pool.tokenDecimals,
+      earnedToken: pool.earnedToken,
+      earnContractAddress: pool.earnContractAddress,
+      status: pool.status,
+      platform: pool.platform,
+      stratType: pool.stratType,
```

Comment:
Do we not need the buy URL, and add liquidity URL?

Reply:
Currently not necessary. If we start consuming from the app, we can add them. Was trying to keep the response smaller until we have a known need to return it.